contracts/core/DaoRegistry.sol

```diff
@@ -386,6 +386,13 @@ contract DaoRegistry is MemberGuard, AdapterGuard {
         emit ExtensionAdded(extensionId, address(extension));
     }
 
+    // v1.0.6 signature
+    function addExtension(
```

Comment:
Added the old signature here so I could call it in Manager. Is there a better way to handle that?